fetch.bs

```diff
@@ -4391,8 +4391,8 @@ steps:
      <a for=request>initiator type</a> is not null, then
      <a for=/>mark resource timing</a> given <var>timingInfo</var>, <var>request</var>'s
      <a for=request>URL</a>, <a for="fetch params">request</a>'s
-     <a for=request>initiator type</a>, <var>global</var>, <var>cacheState</var>, and
-     <var>bodyInfo</var>.
+     <a for=request>initiator type</a>, <var>global</var>, <var>cacheState</var>,
+     <var>bodyInfo</var>, and <var>response</var>'s <a for=response>status</a>.
```

Comment:
I was wrong about this - in the case of iframes the status would leak here. You've reached the same problem in the Chromium implementation, need to think about this :)

Reply:
Updated it to also take into account `request`'s mode and also origin